File: csharp/09_strings.cs
using System;
using System.Linq;

class Program {
    static void Main() {
        string name = "ulas";
        Console.WriteLine(name.ToUpper());
        Console.WriteLine(name.ToLower());
        Console.WriteLine(name.Length);

        int age = 20;
        Console.WriteLine($"{name} is {age} years old");

        Console.WriteLine(name.Substring(0, 2)); // ul

        // reverse without MemoryExtensions ambiguity
        char[] chars = name.ToCharArray();
        Array.Reverse(chars);
        Console.WriteLine(new string(chars));

        Console.WriteLine(name.Contains("ula"));
        Console.WriteLine(name.StartsWith("u"));
    }
}

File: go/09_strings.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	name := "ulas"
	fmt.Println(strings.ToUpper(name))
	fmt.Println(strings.ToLower(name))
	fmt.Println(len(name))

	age := 20
	fmt.Printf("%s is %d years old\n", name, age)

	fmt.Println(name[0:2]) // ul

	fmt.Println(strings.Contains(name, "ula"))
	fmt.Println(strings.HasPrefix(name, "u"))
}
